extensions/survey/src/survey.ts

```diff
@@ -0,0 +1,46 @@
+import { Util } from "@k8slens/extensions";
+import { machineIdSync } from "node-machine-id";
```

Comment:
Shouldn't this also be a dep? Along with `got` and `refiner-js`? Or are those only devDeps because they are compiled into the JS during build?

Reply:
yes those are compiled into the JS